testprojects/src/java/org/pantsbuild/testproject/manifest/BUILD

```diff
@@ -45,5 +45,4 @@ jvm_app(
 java_agent(
   name = 'agent',
   agent_class = 'org.pantsbuild.testproject.manifest.Agent',
-  sources = ['Agent.java'],
```

Comment:
@illicitonion do you recall adding this in https://github.com/pantsbuild/pants/pull/5722? The source file wasn't added and I'm not sure if that was intentional or not.

Reply:
Oops. https://github.com/pantsbuild/pants/pull/8737